contracts/Tokens/VTokens/VToken.sol

```diff
@@ -511,6 +533,43 @@ contract VToken is VTokenInterface, Exponential, TokenErrorReporter {
         return _setInterestRateModelFresh(newInterestRateModel);
     }
 
+    /**
+     * @notice Accrues interest and updates the stable interest rate model using _setStableInterestRateModelFresh
+     * @dev Admin function to accrue interest and update the stable interest rate model
+     * @param newStableInterestRateModel The new interest rate model to use
+     * @return uint 0=success, otherwise a failure (see ErrorReporter.sol for details)
+     * custom:events Emits NewMarketInterestRateModel event; may emit AccrueInterest
+     * custom:events Emits NewMarketStableInterestRateModel, after setting the new stable rate model
+     * custom:access Controlled by AccessControlManager
+     */
+    function setStableInterestRateModel(StableRateModel newStableInterestRateModel) public returns (uint256) {
+        // require(IAccessControlManager(accessControl).isAllowedToCall(msg.sender, "setStableInterestRateModel(address)"), "access denied");
```

Comment:
uncomment the ACL line?

Reply:
Fixed in this [commit](https://github.com/VenusProtocol/venus-protocol/pull/210/commits/46f0cf168db69632d3f6a108f589965f5544c579)